README.md

```diff
@@ -22,35 +22,35 @@ This app demonstrates Rails 7 with PostgreSQL, import maps, turbo, stimulus and
 ## Initial setup
 ```
 cp .env.example .env
-docker compose build
-docker compose run --rm web bin/rails db:setup
+docker-compose build
+docker-compose run --rm web bin/rails db:setup
```

Comment:
`docker compose` is relatively new in Docker Desktop.  What version of Docker are you running?

`docker-compose` is the older version of compose

references: 
* https://docs.docker.com/compose/cli-command/
* https://docs.docker.com/compose/cli-command-compatibility/

Reply:
docker-compose version 1.29.2, build 5becea4c